frontend/src/interfaces/User.ts

```diff
@@ -0,0 +1,8 @@
+export interface User {
+    email: string;
+    password: string;
+    imgUrl?: string;
+    _id?: string;
```

Comment:
`_id` shoud be ` mongoose.Types.ObjectId`

Reply:
done